pkg/cmd/infra/router/template.go

```diff
@@ -126,6 +126,10 @@ type TemplateRouter struct {
 	CaptureHTTPCookie                   *templateplugin.CaptureHTTPCookie
 	HTTPHeaderNameCaseAdjustmentsString string
 	HTTPHeaderNameCaseAdjustments       []templateplugin.HTTPHeaderNameCaseAdjustment
+	HTTPResponseHeadersString           string
```

Comment:
I'd prefer if there were different names for the ingress controller headers vs the route headers, especially when you need to handle these and overrides in the template.  Could you rename the variables to have prefixes `Controller` or `Route` where applicable?

Reply:
These fields in `TemplateRouter` have configuration that ultimately came from the IngressController object.  The route configuration is on the `ServiceAliasConfig` object.  See <https://github.com/openshift/enhancements/blob/master/enhancements/ingress/set-delete-http-headers.md#implementation-detailsnotesconstraints>.